pgx-utils/src/rewriter.rs

```diff
@@ -718,21 +694,18 @@ impl FunctionSignatureRewriter {
                         let mut type_ = crate::sql_entity_graph::UsedType::new(*type_)
                             .unwrap()
                             .resolved_ty;
-                        let is_option = type_matches(&type_, "Option");
 
-                        let ts = if is_option {
-                            let option_type = extract_option_type(&type_);
-                            let mut option_type = syn::parse2::<syn::Type>(option_type).unwrap();
+                        let ts = if let Some(mut option_type) = extract_type_from_option(&type_) {
                             crate::staticize_lifetimes(&mut option_type);
 
                             quote_spanned! {ident.span()=>
-                                let #name = pgx::pg_getarg::<#option_type>(#fcinfo_ident, #i);
+                                let #name : #type_ = pgx::pg_getarg::<#option_type>(#fcinfo_ident, #i);
                             }
                         } else if type_matches(&type_, "pg_sys :: FunctionCallInfo")
                             || type_matches(&type_, "pgx :: pg_sys :: FunctionCallInfo")
                         {
                             quote_spanned! {ident.span()=>
-                                let #name = #fcinfo_ident;
+                                let #name: pgx::pg_sys::FunctionCallInfo  = #fcinfo_ident;
```

Comment:
rustfmt often leaves formatting in macro contexts untouched. Here, `#name:` is used, elsewhere, `#name :` is used, this must be consistent, and preferably consistent with the default type annotation formatting that rustfmt emits.

Reply:
Good catch. I'll update.